content/uc-doc/upgrade/index.md

```diff
@@ -1,7 +1,3 @@
 ---
 title: Upgrading
 ---
-
-- [Upgrading Procedure](/uc-doc/upgrade/introduction)
-- [Upgrade notes](/uc-doc/upgrade/upgrade_notes)
-- [Upgrade to a specific version of Wazo](/uc-doc/upgrade/upgrade_specific_version)
```

Comment:
Theses links, do not looks like `404`

Reply:
No, by AFAICS, they are not visible anywhere, are they?

Reply:
Yes, from here : https://wazo-platform.org/uc-doc/upgrade/ , but well you need to manipulate the URL to have access to it. Not that critical, it's okay 😄